src/esbuild.js

```diff
@@ -16,5 +16,6 @@ esbuild.build({
   minify: AppConstants.NODE_ENV !== 'dev',
   sourcemap: AppConstants.NODE_ENV !== 'dev',
   splitting: true,
-  treeShaking: true
+  treeShaking: true,
+  platform: 'node'
```

Comment:
ESBuild is only bundling for the client, so I think leaving `platform` null is actually preferred, since it defaults to `'browser'`.  

That being said, the docs suggest we might use `platform: 'neutral'` instead of `'browser'` to use more modern ES features.  `'browser'` wraps everything in an iife, which we don't necessarily need/want.
https://esbuild.github.io/api/#platform

Reply:
I think I added this in order to get jest working. I'm still in the process of fixing jest to run tests within `/src`... it seems like a long process